private_sharing/api_views.py

```diff
@@ -256,11 +259,22 @@ class ProjectFileDirectUploadCompletionView(ProjectFormBaseView):
     form_class = DirectUploadDataFileCompletionForm
 
     def post(self, request):
-        super(ProjectFileDirectUploadCompletionView, self).post(request)
+        super().post(request)
 
         data_file = ProjectDataFile.all_objects.get(
             pk=self.form.cleaned_data['file_id'])
 
+        s3 = boto3.resource('s3')
+
+        try:
+            s3.Object(bucket, data_file.parent.file.name).load()
```

Comment:
A note before closing this - the code as written was failing because `bucket` wasn't defined.